src/pages/HomePage.tsx

```diff
@@ -12,6 +13,7 @@ const HomePage = () => (
       link={tempEventCardData.link}
       title={tempEventCardData.title}
     />
+    <Footer />
```

Comment:
IMO we need to create a layout  with header and footer, so we can reuse that

Reply:
Creating as an issue